src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/StackFrameInfo.cs

```diff
@@ -1,54 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Ugh. With all the hacks, erm... workarounds, we're going to have to do to maintain binary compat, keeping a class around that no one will ever use (or indeed *has* ever used looking at the git history) is pretty minor I guess.

Presumably our tools for checking binary compat will yell at us later if we remove it now.

@Eilon Do we want to file issues for all the easy-to-make breaking changes we want to make in the next major release? Should we create a tag for this?